include/internal/packet.h

```diff
@@ -633,7 +633,7 @@ struct wpacket_st {
     /* Number of bytes written so far */
     size_t written;
 
-    /* Maximum number of bytes we will allow to be written to this WPACKET */
+    /* Maximum number of bytes we will allow one to be written to this WPACKET */
```

Comment:
This looks like an incorrect change.  _Maximum number of allowed bytes to write to this WPACKET_ ?